benchmarks/transformer/score_mod.py

```diff
@@ -263,12 +293,10 @@ def get_average_speedups(results: List[Experiment], type: str):
     min_config_dict = results[min_speedup_index].config.asdict()
 
     # Extract function names from score_mod strings
-    max_config_dict["score_mod"] = (
-        max_config_dict["score_mod"].__name__.split("<locals>.")[-1].split(" at ")[0]
-    )
-    min_config_dict["score_mod"] = (
-        min_config_dict["score_mod"].__name__.split("<locals>.")[-1].split(" at ")[0]
-    )
+    max_config_dict["score_mod"] = get_func_name(max_config_dict["score_mod"])
+    max_config_dict["mask_mod"] = get_func_name(max_config_dict["mask_mod"])
```

Comment:
I actually just landed a PR to do the same thing: https://github.com/pytorch/pytorch/pull/132424